### ISSUE TITLE
Cannot access ambient const enums when the '--isolatedModules' flag is provided

### ISSUE THREAD

**Issue by luaneko** [naptha/tesseract.js#434]
**Describe the bug**
This error occurs when attempting to use tesseract.js typescript enums which are defined as `const enum`. This prevents these enums from being used in popular frameworks such as **create-react-app**.

Specifically: `OEM` and `PSM` enums.

**To Reproduce**
1. Set up a create-react-app project and install tesseract.js.
2. Attempt to initialize a worker with an OEM mode.

**Expected behavior**
Code compiles.

**Screenshots**
![image](https://user-images.githubusercontent.com/27663662/78453354-db28fa80-76dc-11ea-8fcd-597e7dac159d.png)

**Desktop (please complete the following information):**
Inapplicable

**Smartphone (please complete the following information):**
Inapplicable

**Additional context**
Possible solution is to remove `const` from these enums.

**Comment by phosphene47** [comment]
I am aware of simply using integers as these are const enums. However since this library is providing an enum anyway, referring to enums should be preferred over integer magic numbers.

**Comment by jeromewu** [comment]
Can you share the full example code to reproduce this issue?

**Comment by Dullson** [comment]
Hello.
It has been almost 2 years but this problem still persists. Reported reproduction steps look good to me but if you still need an example, I have created a project using [react example from readme](https://github.com/jeromewu/tesseract.js-react-app) and added typescript. [My reproduction example.](https://github.com/Dullson/tesseract.js-react-app--434-repro)
The only change I have made to the template code is adding  `
await worker.setParameters({
      tessedit_pageseg_mode: PSM.AUTO,
    })
` to demonstrate `const enum` issue.
[Typescript has an explanation on the topic.](https://www.typescriptlang.org/docs/handbook/enums.html#const-enum-pitfalls)
For the time being, I removed `const` from those two enums and got no problems so far.
